doc/porting/kconfig.rst

```diff
@@ -0,0 +1,28 @@
+.. _kconfig:
```

Comment:
this should go into doc/tools/kconfig/..

Reply:
Moved it there.